net/socket/udp_socket_starboard.cc

```diff
@@ -778,8 +744,8 @@ void UDPSocketStarboard::PostSendBuffers() {
            << write_async_outstanding_ << " total";
   SbSocketAddress sb_address;
   DCHECK(remote_address_.get()->ToSbSocketAddress(&sb_address));
-  base::PostTaskAndReplyWithResult(
-      GetTaskRunner(), FROM_HERE,
+  base::ThreadPool::PostTaskAndReplyWithResult(
```

Comment:
I think this changes the behavior a little, because previously it would use one sequenced task runner and now it's just posting to the threadpool, which means that previously the posted tasks would be guaranteed to execute in order, and now there is no ordering guarantee. So if there is a following call to `PostSendBuffers`, then the task posted for that call may sometimes actually run before this one. 

Wouldn't that possibly change the order of the data being sent resulting in corruption?

Reply:
Fixed.